datacube_stats/output_drivers.py

```diff
@@ -366,6 +366,7 @@ class GeotiffOutputDriver(OutputDriver):
 
     Con write all statistics to the same output file, or each statistic to a different file.
     """
+    _driver_name = 'Geotiff'
```

Comment:
Data cube core seems to have 'GeoTiff' spelling for this

Reply:
It's a bit inconsistent there too. DEA is universally GeoTiff, but some parts of ODC use GeoTIFF.

GeoTiff is good enough, and who really cares anyway. :)